docs-ref-conceptual/whats-new-overview.md

```diff
@@ -1,18 +1,25 @@
 ---
-title: What's new in Azure Command-Line Interface (CLI) | Microsoft Docs
-description: Learn what's new in the Azure CLI
+title: Azure CLI release highlights | Microsoft Docs
+description: Learn what's new in the Azure CLI. Read about new features and upcoming changes. Check here for announcements.
 author: dbradish-microsoft
 ms.author: dbradish
-ms.date: 05/21/2024
-ms.topic: conceptual
+ms.date: 08/01/2024
+ms.topic: concept-article
 ms.service: azure-cli
-ms.tool: azure-cli 
 ms.custom: devx-track-azurecli
 keywords: Azure CLI, new articles, new references, new samples, announcements
+#customer intent: As an Azure CLI developer, I want an easy way to see product change highlights and upcoming breaking change announcements.
 ---
-# What's new in the Azure CLI
 
-This page highlights new features, articles and learning paths for the Azure CLI.
+# Azure CLI release highlights
+
+This page highlights new features and upcoming changes for the Azure CLI.
+
+## Docker container image
+
+Beginning with Azure CLI version 2.67.0, to be released on November 21, 2024, the Docker container for the Azure CLI will be based on Azure Linux, formerly known as [CBL-Mariner](https://mcr.microsoft.com/en-us/product/cbl-mariner/base/core/about). When running the Azure CLI in a Docker container, always use a tag, like `cbl-mariner2.0` and do not rely on `latest`.
```

Comment:
Do we need to explicitly tell the customers that the latest tag, which was based on Alpine, will be changed to Azure Linux?

Reply:
In my opinion, if the user only runs the `az` command, the `latest` tag is sufficient. However, if they need to run platform-dependent commands, a specific tag is required.

Reply:
We need to be clear with customers about the change since any dependencies they have on the image could break,

Reply:
@dcaro and @bebound, This conversation looks resolved so I am marking it as such.